packages/pocket-ic/src/common/rest.rs

```diff
@@ -546,39 +546,48 @@ impl From<SubnetConfigSet> for ExtendedSubnetConfigSet {
     }
 }
 
-/// Forward-compatible configuration type used instead of `bool` and `Option<bool>`:
-/// if provided, the corresponding feature is enabled.
 #[derive(Debug, Clone, Eq, Hash, PartialEq, Serialize, Deserialize, Default, JsonSchema)]
-pub struct EmptyConfig {}
+pub enum NonmainnetFeaturesConfig {
+    #[default]
+    Mainnet,
+    Disabled,
+    Enabled,
+}
```

Comment:
What is the purpose of having the three `Mainnet`, `Enabled` and `Disabled` variants plus an `Option` for each field?

How about we simplify the enum:

```suggestion
pub enum NonmainnetFeaturesConfig {
    Enabled,
    Disabled,
}
```

And then for each field, we manage the mainnet/default manually?

```rust
beta_features.clone().unwrap_or(NonmainnetFeaturesConfig::Disabled)
```